numpy/distutils/system_info.py

```diff
@@ -1766,10 +1748,18 @@ def get_atlas_version(**config):
 
 class lapack_opt_info(system_info):
     notfounderror = LapackNotFoundError
+
     # List of all known LAPACK libraries, in the default order
-    lapack_order = ['mkl', 'openblas', 'flame', 'atlas', 'lapack']
+    lapack_order = ['accelerate', 'mkl', 'openblas', 'flame', 'atlas', 'lapack']
```

Comment:
I think the order should be:
```
['mkl', 'openblas', 'flame', 'accelerate', 'atlas', 'lapack']
```
If any of the first three are installed, they should be preferred over Accelerate imho, since they're more up-to-date and offer competitive performance.

Reply:
And same for `blas_order`